.craft.yml

```diff
@@ -32,6 +32,10 @@ targets:
   - name: npm
     id: '@sentry-internal/feedback'
     includeNames: /^sentry-internal-feedback-\d.*\.tgz$/
+  ## 1.8 ReplayCanvas package (browser only)
```

Comment:
Do we need to publish this? Or could we just inline this into browser/replay, same as we do with e.g. replay-worker? IMHO if we can avoid to publish it, we should!

Reply:
@mydea I added this back, it's been a pain trying to test this package without it being published. e.g. in my test app, I'll use `yalc` to add `@sentry/browser @sentry/replay`, but yarn will complain:

```
error Couldn't find package "@sentry-internal/replay-canvas@7.93.0" required by "@sentry/browser@file:.yalc/@sentry/browser" on the "npm" registry.
```